src/pages/Controllers/ControllerChart/ControllerChart.test.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { mount } from "enzyme";
+import { MemoryRouter } from "react-router";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+
+import ControllerChart from "./ControllerChart";
+
+const mockStore = configureStore([]);
+
+describe("Controllers chart", () => {
+  it("renders if totals correctly", () => {
+    const store = mockStore({
```

Comment:
These `store`s aren't required nor is the `MemoryRouter` or `Provider` for any of the tests for this component.